package.json

```diff
@@ -6,21 +6,24 @@
   "scripts": {
     "start": " cd views && npm start",
     "build": "cd views && npm run build",
-    "test": "cd views && npm run test "
+    "test": "cd views && npm run test ",
+    "lint": "eslint *; exit 0"
   },
```

Comment:
Use a `.`, not a `*`. `*` picks up on `.json` and `.md` files which we don't want to lint.

Reply:
got it